es5.js

```diff
@@ -0,0 +1,74 @@
+function BuilderES5 (str) {
+    this.str = str;
+}
+
+function StringBuilder (str) {
+    BuilderES5.call(this, str);
+}
+    
+StringBuilder.prototype.plus = function (...str) {
+    this.str = [this.str, ...arguments].join('');
+    return this.str;
+}
+
+StringBuilder.prototype.minus = function (n) {
+    if (!this.str) {
+        throw "String is empty!";
+    } else {
+        this.str = this.str.slice(0, this.str.length - n);
+        return this.str;
+    }
+}
+
+StringBuilder.prototype.multiply = function (n) {
+    if (!this.str) {
+        throw "String is empty!";
+    } else {
+        let subStr = '';
+        for (let i = 0; i < n; i++) {
+            subStr += this.str;
+        }
+        this.str = subStr;
+        return this.str;
+    }
+}
+
+StringBuilder.prototype.divide = function (n) {
+    if (!this.str) {
+        throw "String is empty!";
```

Comment:
I like this check, but there is a repetition. You can make this an internal method that you'd use within other methods that would throw error if string is empty